documentation/develop.md

```diff
@@ -0,0 +1,123 @@
+# Development
+
+Please as a first start, tell the community about your intent to develop a feature or fix a bug. Search for the associated issue if it exists or open one with steps to reproduce (for bugs) or a [user story](https://en.wikipedia.org/wiki/User_story#Principle) (for features).
+
+## Setup
+
+### Prerequisites
+
+To setup your environment, you would need to install the following dependencies:
+ - git
+ - [nvm](https://github.com/nvm-sh/nvm/blob/master/README.md) (recommanded) or nodejs installed on your system
+ - Firefox and/or Chromium to run the end-to-end (browser) tests
+ - Python (preferably Python 3.9) and virtualenv
+    - :warning: As of 2023-06-06, Python 3.11 is not supported due to the version of the [wrapt dependency](https://github.com/GrahamDumpleton/wrapt/issues/196)
+
+### Clone the repository
+
+```bash
+$ git clone https://github.com/gristlabs/grist-core
+```
+
+And then, enter in the grist-core root directory:
```

Comment:
"enter into" or (better) just "enter" I'd say (sorry for constant nitpicking here)

Reply:
No that's OK!